### PR TITLE
Re-working the README for relevance

### PR DESCRIPTION
**Pull request by kenperkins**
Still need to figure out docs for windows users, and precedence.

### CODE REVIEW

**Comment by kenperkins** [comment]
I believe this is now Ready for review.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,33 @@
 # rack
-An opinionated CLI for Rackspace interactions
+The official command line interface for the Rackspace Cloud.
+
+`rack` provides a consistent interface for interacting with the Rackspace Cloud. For example, creating a new server:
+
+```
+$ rack servers instance create
```
Comment (jrperritt) [inline README.md]: Can we add some `\` at the end of the lines of these commands for users who try to copy-and-paste it?

Comment (rgbkrk) [inline README.md]: Ditto on that. These should run as if this was a literate programming context.

Comment (kenperkins) [inline README.md]: Good call, I'll fix.

**Comment by kenperkins** [comment]
Also realize it's missing the windows section.

**Comment by jrperritt** [comment]
ping @kenperkins

**Comment by kenperkins** [comment]
Good to go.

**Comment by rgbkrk** [comment]
:shipit:

**Pull request merged**